atomic_reactor/plugins/post_pulp_tag.py

```diff
@@ -53,12 +54,13 @@ def __init__(self, tasker, workflow, pulp_registry_name=None, pulp_secret_path=N
     def set_v1_tags(self, v1_image_id):
         image_names = self.workflow.tag_conf.images[:]
         handler = get_pulp_session(self.workflow, self.log, self.pulp_fallback)
-
-        pulp_repos = handler.create_dockpulp_and_repos(image_names)
+        handler.create_dockpulp()
+        pulp_repos = handler.create_repos_mapping(image_names)
```

Comment:
If you don't want to run _ensure_repos, I'd rather pass an optional argument to create_dockpulp_and_repos called "ensure" that defaults to True, and runs  _ensure_repos if it is True. Just seems tidier than breaking it up like this.